src/app/space/analyze/analyze-overview/analyze-overview.component.spec.ts

```diff
@@ -0,0 +1,135 @@
+import { AnalyzeOverviewComponent } from './analyze-overview.component';
+
+import {
+  Component,
+  DebugElement,
+  Input,
+  NO_ERRORS_SCHEMA
+} from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { Broadcaster } from 'ngx-base';
+import { BsModalService } from 'ngx-bootstrap/modal';
+import { Context, Contexts, Space } from 'ngx-fabric8-wit';
+import { AuthenticationService, User, UserService } from 'ngx-login-client';
+import { ConnectableObservable, Subscription } from 'rxjs';
+import { Observable, Subject } from 'rxjs';
+import { createMock } from 'testing/mock';
+import {
+  initContext,
+  TestContext
+} from 'testing/test-context';
+
+@Component({
+  template: '<alm-analyzeOverview></alm-analyzeOverview>'
+})
+class HostComponent { }
+
+fdescribe('AnalyzeOverviewComponent', () => {
+  type TestingContext = TestContext<AnalyzeOverviewComponent, HostComponent>;
+
+  let modalService: jasmine.SpyObj<BsModalService>;
+  let broadcaster: jasmine.SpyObj<Broadcaster>;
+  let authentication: jasmine.SpyObj<AuthenticationService>;
+  let ctxSubj: Subject<Context> = new Subject<Context>();
+  let fakeUserObs: ConnectableObservable<User> = Observable.of({
+    id: 'loggedInUser'
+  } as User).publish() as ConnectableObservable<User>;
+
+  initContext(AnalyzeOverviewComponent, HostComponent, {
+    providers: [
+      { provide: BsModalService, useFactory: (): jasmine.SpyObj<BsModalService> => createMock(BsModalService) },
+      { provide: Broadcaster, useFactory: (): jasmine.SpyObj<Broadcaster> => createMock(Broadcaster) },
+      { provide: AuthenticationService, useValue: ({ isLoggedIn: () => true }) },
+      { provide: Contexts, useValue: ({ current: ctxSubj }) },
+      { provide: UserService, useValue: ({ loggedInUser: fakeUserObs }) }
+    ],
+    schemas: [
+      NO_ERRORS_SCHEMA
+    ]
+  });
+
+  it('should call to check the user space', function(this: TestingContext) {
+    spyOn(this.testedDirective, 'userOwnsSpace');
+
+    ctxSubj.next({
+      space: {
+        relationships: {
+          'owned-by': {
+            data: {
+              id: 'loggedInUser'
+            }
+          }
+        }
+      } as Space
+    } as Context);
+
+    this.detectChanges();
+    expect(this.testedDirective.userOwnsSpace).toHaveBeenCalled();
+  });
+
+  it('should disable the button if user service unavailable', function(this: TestingContext) {
+    fakeUserObs = null;
```

Comment:
I don't think setting the observable reference to null here is going to do anything - the component accesses this observable via the UserService it receives via DI when constructed. I think emitting a falsey value from this observable - same as you've done with ctxSubject below - should do what was intended here.